client/app/src/main/java/com/example/mrides/controller/RequestHandler.java

```diff
@@ -65,6 +70,9 @@ public String getBodyContentType() {
     }
 
     public void getStringRequest(String url, Context context){
```

Comment:
Avoid `get-` prefix for void method. It can be confusing with JavaBeans specification on accessor and mutator methods.